app/views/associate_ducr_summary.scala.html

```diff
@@ -29,22 +29,37 @@
         @components.back_link(consolidations.routes.AssociateDucrController.displayPage())
 
         @components.page_title(
-            title = Some(messages("associate.ucr.summary.title", mucr)),
-            subtitle = Some(messages("associate.ucr.summary.hint"))
+            title = Some(messages("associate.ucr.summary.title"))
         )
 
+        <h2 id="add-consignment">@messages("associate.ucr.summary.addConsignment")</h2>
```

Comment:
First time I am seeing these `h2`s

Reply:
In a lot of pages in movements we have this h2. Hmm probably it's good moment to create a component for it, but name `heading-2` is stupid :D